pkgs/by-name/ce/celeste-classic/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+, autoPatchelfHook
+, SDL2
+, practiceMod ? false
+}:
+
+let
+  directory = if practiceMod then "CELESTE*Practice*" else "CELESTE";
+  srcbin = if practiceMod then "celeste_practice_mod" else "celeste";
+  outbin = if practiceMod then "celeste-classic-pm" else "celeste-classic";
+in stdenvNoCC.mkDerivation {
```

imo, breaking the line here would look less cluttered
```suggestion
in
stdenvNoCC.mkDerivation {
```